### ISSUE TITLE
No gap between pages

### ISSUE THREAD

**Issue by ElModdy** [xournalpp/xournalpp#3633]
**Is your feature request related to a problem? Please describe.**
I use xournal to annotate pdfs.
Many pdfs are generated by note-taking applications that don't have the concept of pages, so they simply cut into pages.
The problem is, it's hard to follow what's near the end of a page because the default gap is too large.
I would like to remove it as if it were a single long page.

**Describe the solution you'd like**
I have noticed this [variable](https://github.com/xournalpp/xournalpp/blob/f72fe7c85bf6ac0525350362b946e10df7a9371c/src/core/gui/Layout.cpp#L27) which may be what I am looking for.
If you could modify it in some way with, for example, a plugin, that would be great.

**Describe alternatives you've considered**
Maybe changing the value and rebuilding the application might be a solution, but I have no experience with this kind of thing
and afraid of making a mess.

**Comment by rolandlo** [comment]
Removing gaps between pages, so that it looks like infinity pages, is problematic, since the pages will still act like single pages. 
I would advise to change the constant you indicated and build from source. If you are on Linux, this is quite straightforward following the steps (from `Install Dependencies` and `Compiling`) in the [wiki](https://github.com/xournalpp/xournalpp/blob/master/readme/LinuxBuild.md). You can just run the built executable from the terminal and don't need to install it on your system. So you can't really mix up.
